mage/main.go

```diff
@@ -35,8 +35,8 @@ const initFile = "magefile.go"
 
 var (
 	force, verbose, list, help, mageInit, keep, showVersion bool
-
-	timestamp, commitHash, gitTag string
+	timeout                                                 int
```

Comment:
you can put spaces between the lines os the types aren't so crazily right justified :)